webui/react/src/components/Searches/Searches.tsx

```diff
@@ -676,6 +676,7 @@ const Searches: React.FC<Props> = ({ project }) => {
               currentColumn.displayName || currentColumn.column,
               settings.columnWidths[currentColumn.column],
               dataPath,
+              undefined,
```

Comment:
shouldn't you be able to remove this argument without issue? `heatmapProps` is marked as optional and behind datapath.